### DISCUSSION TITLE
Cfg for Deno

### DISCUSSION THREAD

**Discussion by BlackAsLight** [denoland/deno#15244] [Ideas]
So I was wondering if a Cfg, like Rust has, for Deno would be possible. I'm trying to write some code for both Deno and the browser, but to work in both it needs different imports, with the rest being the exact same. When bundling though, it would be nice if I could exclude certain imports and functions so they don't get added with to never be ran.

**Comment by zandaqo** [comment]
https://github.com/denoland/dnt

**Reply by BlackAsLight** [reply]
Not what I was asking but I found a solution to what I was trying.
